ci-scripts/publish.sh

```diff
@@ -8,7 +8,11 @@ git config --global user.name "fundamental-bot"
 # delete temp branch
 git push "https://$GH_TOKEN@github.com/$TRAVIS_REPO_SLUG" ":$TRAVIS_BRANCH" > /dev/null 2>&1;
 
-cd library
+# Frank Note:
+# library package.json should not have any dependencies or dev dependencies, only peer.
+# This should install from the root. s
```

Comment:
we are publishing a npm package which is just the library, not the whole repo. That's why we are running it in the library

Reply:
Yes understood, but all its is ok. But you dont need to have dependencies and devDep, it shoudl work fine with the global ones, and you use those globals ones just to compile you lib into DIST folder, and then npm publish takes it from there. Once you install a library, then those 2 sections became absolete as they are not used at all.